app/utilities/featureToggleParser/featureToggleParser.factory.spec.js

```diff
@@ -2,8 +2,303 @@
 
 describe('featureToggleParser', function () {
 
-  it('TBD',function() {
+  beforeEach(module('myApp.utilities'));
 
+  var featureToggleParser;
+  
+  beforeEach(inject(function(_featureToggleParser_) {
+  
+      featureToggleParser = _featureToggleParser_;
+  
+  }));
+  
+  it('should featureToggleParser be defined', function() {
+  
+      expect(featureToggleParser).toBeDefined();
+  
+  });
+  
+  it('should featureToggleParser.parseConfigObject() be defined', function() {
+  
+      expect(featureToggleParser.parseConfigObject).toBeDefined();
+  
+  });
+  
+  it('should featureToggleParser.parseToggleObject() be defined', function() {
+  
+      expect(featureToggleParser.parseToggleObject).toBeDefined();
+  
+  });
+  
+  describe('parseConfigObject()', function() {
+  
+      var result, newConfig;
+  
+      it("should return false for enableDates when the parseConfigObject is called with enableDates as ['false', 'off', '0', 'no']", function() {
+  
```

Comment:
This test only verifies a single value. This should be:

'should set enableDates to false for input of enableDates = "no"'